backend/src/db/repositories/TranslationRepository.ts

```diff
@@ -0,0 +1,26 @@
+import DB from '../DB';
+import {TranslationRaw} from '../types/TranslationRaw';
+
+export default class TranslationRepository {
+    private db: DB;
+
+    constructor(db: DB) {
+        this.db = db;
+    }
+
+    async getTranslation(content_source_id: number, language: string): Promise<TranslationRaw | undefined> {
+        return await this.db.fetchOne<TranslationRaw>('select * from translations where content_source_id=:content_source_id and language=:language', {
```

Comment:
is it possible to have multiple translations for a given content source item? If not, comparing `language` is redundant

Reply:
Not at the moment, but the db design (composite key: content_source_id, language) allows to add this fairly easy. For example, if we decide to allow users to select their preferred translation language in the future.

But I agree, it's redundant at the moment, not sure if it's worth deleting though (somewhat invasive change).